civis/tests/test_io.py

```diff
@@ -398,6 +398,242 @@ def test_civis_file_to_table_table_doesnt_exist(
             **expected_kwargs
         )
 
+    @pytest.mark.civis_file_to_table
+    @mock.patch('civis.io._tables._process_cleaning_results')
+    @mock.patch('civis.io._tables._run_cleaning')
+    def test_civis_file_to_table_table_doesnt_exist_valid_table_columns_provided(
```

Comment:
I added a test to show that when the table does not exist, but all sql_types are provided, that even if they are different, it does not throw an error and evaluates them correctly. No cleaning happens.